app/models/feed.rb

```diff
@@ -181,7 +181,7 @@ def saved_search_was
   def self.save_request(feed_id, type, query, webhook_url, result_ids)
     media = Request.get_media_from_query(type, query, feed_id)
     request = Request.create!(feed_id: feed_id, request_type: type, content: query, webhook_url: webhook_url, media: media, skip_check_ability: true)
-    request.attach_to_similar_request!
+    # request.attach_to_similar_request!
```

Comment:
I don't want to remove `send_to_alegre` in its entirety. It still works fine for text. I feel we should only comment the one part of the function that tries to send media, but leave everything else as is.